manifests/params.pp

```diff
@@ -354,7 +381,7 @@
     $secpcrematchlimit = 1500
     $secpcrematchlimitrecursion = 1500
     $modsec_secruleengine = 'On'
-    if $::operatingsystem == 'Debian' and versioncmp($::operatingsystemrelease, '9') >= 0 {
+    if ($::operatingsystem == 'Debian' and versioncmp($::operatingsystemrelease, '9') >= 0) or ($::operatingsystem == 'Ubuntu' and versioncmp($::operatingsystemrelease, '18.04') >= 0) {
```

Comment:
Makes me think a helper function in stdlib could useful: `if os_version_gt('Debian', '9') or os_version_gt('Ubuntu', '18.04') {`. Might write up a patch for that.

Reply:
https://github.com/puppetlabs/puppetlabs-stdlib/pull/972